lib/order.rb

```diff
@@ -0,0 +1,17 @@
+
+class Order
+  attr_reader :current_order
+
+  def initialize(menu)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `menu`. If it's necessary, use `_` or `_menu` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)